tests/e2e_tests/integration/07-saml.spec.ts

```diff
@@ -0,0 +1,183 @@
+
+      await expect(page).toHaveURL(`${baseUrl}ui/settings/organization-and-billing`);
+      // Check input[type="checkbox"]
+      await page.locator('input[type="checkbox"]').check();
+      // Click text=input with the text editor
+      await page.locator('text=input with the text editor').click();
+      // Press Tab
+      await page.locator('text=SAML metadataImport from a fileEnter to Rename, ⇧Enter to PreviewCancelSave').press('Tab');
+      // Press Tab
+      await page.locator('text=SAML metadataImport from a file >> div[role="presentation"]').press('Tab');
+      // Press Tab
+      await page.locator('text=Import from a file').press('Tab');
+      // Press Tab
+      await page.locator('text=SAML metadataImport from a file >> button').nth(1).press('Tab');
+      console.log('typing metadata');
+      await page.locator('[aria-label="Editor content\\;Press Alt\\+F1 for Accessibility Options\\."]').type(metadata.replace(/(?:\r\n|\r|\n)/g,''))
+      console.log('typing metadata done.');
+
+      await page.screenshot({'path':'saml-edit-saving.png'});
+      // Click text=Save >> nth=1
+      await page.locator('text=Save').nth(1).click();
+      await page.screenshot({'path':'saml-edit-saved.png'});
+      console.log('typing metadata done. waiting for Entity ID to be present on page.');
+      await page.waitForSelector('text=Entity ID');
+      console.log('waiting over continuing.');
+
+      const response = await axios({
+        url: `${baseUrl}api/management/v1/useradm/sso/idp/metadata`,
+        method: 'GET',
+        headers: {
+          Authorization: `Bearer ${jwt}`,
+          'Content-Type': 'application/json'
+        },
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        })
+      });
+
+      if (response.status !== 200) {
+        throw 'expected 201';
+      }
+
+      const id=response.data[0]['id'];
+      console.log(`got metadata id: ${id}`)
+
+      const expected_login_url=`${baseUrl}api/management/v1/useradm/auth/sso/${id}/login`;
+      const find_results = await page.locator('span',{'hasText' : expected_login_url})
+      await expect(find_results).toHaveText(expected_login_url);
+      console.log(`got span with login url: ${await find_results.count()}`)
+
+      const expected_acs_url=`${baseUrl}api/management/v1/useradm/auth/sso/${id}/acs`;
+      const find_results_acs = await page.locator('span',{'hasText' : expected_acs_url})
+      await expect(find_results_acs).toHaveText(expected_acs_url);
+      console.log(`got span with acs url: ${await find_results_acs.count()}`)
+
+      const expected_sp_meta_url=`${baseUrl}api/management/v1/useradm/sso/sp/metadata/${id}`;
+      const find_results_sp_meta = await page.locator('span',{'hasText' : expected_sp_meta_url})
+      await expect(find_results_sp_meta).toHaveText(expected_sp_meta_url);
+      console.log(`got span with SP metadata url: ${await find_results_sp_meta.count()}`)
```

Comment:
this seems to duplicate the metadata checks the final login process does... + if you want to use the obtained values, they are available in the UI too...

Reply:
not entirely: these links you want to have to copy-paste to your identity provider,  so this checks that we present them correctly.

Reply:
yes, but the links the UI shows are taken straight from the metadata just like you show them here - if either the link we use to login (that could be copied and used to login the newly created user) or any of the values shown in the ui (that are derived from the metadata) don't align, the login won't work. The link creation in the UI itself is tested through unit tests already.

Reply:
okey, no problema, so you vote for removing those three blocks?